android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -228,19 +228,18 @@ public static Boolean responseRequiresJSInjection(Response response) {
     final int responseCode = response.code();
 
     String responseBody = "";
-    String contentTypeAndCharset = "";
 
     try {
       assert response.body() != null;
-      responseBody = response.body().string();
+      responseBody = response.peekBody(Long.MAX_VALUE).string();
```

Comment:
I assume this change was made to for performance reasons? The documentation for `peekBody` recommends a smaller value, and I think it would suffice for our purposes, too: https://square.github.io/okhttp/4.x/okhttp/okhttp3/-response/peek-body/